index.js

```diff
@@ -42,6 +46,7 @@ module.exports = {
     "max-line-length": [ 80, {
       ignore: "non-comments",
     } ],
+    "max-nesting-depth": 3,
```

Comment:
So, having actually 4 levels (due to stylelint mechanics) is ok with you guys? Sorry again, I don't nest more than 2 levels myslelf, but [seing this](https://github.com/suitcss/stylelint-config-suitcss/commit/f1e85f6c84539f3ab0816e2362097ba97460bc50#diff-518498ef0535a2e4c194b95d1cff9c3fR25) started to worry a bit.

> Wouldn't it make sense to keep it all in one place?

Maybe [something like this](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#bemdepth) was meant. Right now stylelint don't even have a rule to limit selectors depths (going to propose one).